specs/Spec_for_adding_user.adoc

```diff
@@ -0,0 +1,166 @@
+# User Management: Adding new User in Tendrl
+
+## Problem description
+
+This spec talks about Adding/Creating a new User to Tendrl by the Admin
+
+## Use Cases
+
+The flow goes as follows:-
+
+* A user login as Admin
+* Clicks on Admin tab in Navigation
+* The navigation drops down
+* Clicks on *Users*
+* Clicks on *Add* button in Users page view
+
+User will land onto https://redhat.invisionapp.com/share/KNB25OEQT#/screens/226063802[Add Users] page
+
+## Proposed change
+
+### UI changes
+
+* Add *Users* tab in Admin navigation tab
+* Add *Add User* Button in Users Page view
+* Add a form for adding a user. The form will contain following feilds:-
+** User ID
+*** Input type : text
+*** Validations:
+**** Should not contain special characters
+**** Should be Unique
+**** Can be of a particular Maximum length
+** First Name
+*** Input type : text
+*** Validations:
+**** Should not contain special characters
+**** Can be of a particular Maximum length
+** Last Name
+*** Input type : text
+*** Validations:
+**** Should not contain special characters
+**** Can be of a particular Maximum length
+** Email
+*** Input type : email
+*** Validations:
+**** Validate email ID
+** Email Notification
+*** Input type : checkbox
+** Password
+*** Input type : password
+** Confirm Password
+*** Input type : password
+** Set a Role
+*** Admin
+*** Normal user
+*** Limited user
+*** Input type : Radio button
+
+
+### Impacted Modules:
+
+#### Tendrl API impact:
+
+```
+POST /api/1.0/users - creates a new user
+```
+
+* Sample API response now : 
+
+```
+{
+	"email": "thardy@tendrl.org",
```

Comment:
Initially, we just had ```name``` field, now according to new mockup https://redhat.invisionapp.com/share/KNB25OEQT#/screens/226063802, we require separate first name and last name